conf/hbavss.local.ini

```diff
@@ -3,6 +3,8 @@
 [general]
 N: 4
 t: 1
+k: 10
+delta: -9999
```

Comment:
what is delta? can the config file have a comment explaining it?

Reply:
delta is something I needed to add to make hbavss work locally, its something AWSConfig uses, @smkuls is probably using this in butterfly netowrk code. I am unsure of its use